README.md

```diff
@@ -51,12 +52,6 @@ Let us assume that we wrote a nice little widget called `MessageCtrl'`. and we w
       "readme": "https://raw.githubusercontent.com/ismail-yilmaz/MessageCtrl/main/README.md"
     }
 ```
-Once these changes are validated and merged, the package(s) will be visible and available for download via the UppHub package manger in our integrated development environment, [TheIDE](https://www.ultimatepp.org/app$ide$GettingStarted_en-us.html):
+Once these changes are validated and merged, the package(s) will be visible and available for download via the UppHub package manager in our integrated development environment, [TheIDE](https://www.ultimatepp.org/app$ide$GettingStarted_en-us.html):
 
-<p align="center">
```

Comment:
Let's stay with the image. I found that repositories with images are more catchy. If you would like to take another better photo just replace the current one.

Reply:
@klugier 
Image is already there, we just don't need to use html tags. It is displayed as usual.

Reply:
Sure, I didn't notice the line below. Thanks for clarification!